es/lockwise-ios.xliff

```diff
@@ -94,23 +98,35 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="10.2.1" build-num="10E1001"/>
     </header>
     <body>
+      <trans-unit id="ehm-2v-Rbb.accessibilityLabel">
+        <source>Firefox Lockwise</source>
+        <note>Class = "UIImageView"; accessibilityLabel = "Firefox Lockwise"; ObjectID = "ehm-2v-Rbb"; Note = "Text logo of the product name for screenreaders";</note>
+      </trans-unit>
       <trans-unit id="hwO-wS-oS4.text">
         <source>Take your passwords everywhere</source>
         <target>Lleva tus contraseñas a todos lados</target>
         <note>Class = "UILabel"; text = "Take your passwords everywhere"; ObjectID = "hwO-wS-oS4";</note>
       </trans-unit>
     </body>
   </file>
-  <file original="lockbox-ios/Common/Resources/InfoPlist.strings" source-language="en" target-language="es" datatype="plaintext">
+  <file original="lockbox-ios/Common/Resources/Strings/InfoPlist.strings" source-language="en" target-language="es" datatype="plaintext">
```

Comment:
This is due to the change found/discussed at #5

Reply:
@Delphine , this change is gonna drop the localizations that currently exist for this segment, notably, this is all the removed `<target>Lockwise</target>` lines like https://github.com/mozilla-l10n/lockwiseios-l10n/pull/6/files#diff-5a7f14dd031edb60ff7fffcf4890f0e2L111 in the attached files.

I think that's OK, as this particular string falls back to correct English brand name anyway, but I'd like you to confirm that call.

Reply:
Agreed it's ok   👍